packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -109,10 +108,10 @@ const BlocksEditor = React.forwardRef(
             initialValue={value || [{ type: 'paragraph', children: [{ type: 'text', text: '' }] }]}
             onChange={handleSlateChange}
           >
-            <InputWrapper direction="column" alignItems="flex-start">
+            <InputWrapper direction="column" alignItems="flex-start" height="512px">
               <BlocksToolbar disabled={disabled} />
               <EditorDivider width="100%" />
-              <Wrapper>
+              <Wrapper grow={1}>
```

Comment:
How come you need the grow 🤔? When I remove it seems to function the same

Reply:
Without the grow, hovering the empty part of the editor does not get the editing cursor, and clicking does not focus it

Reply:
Got it thanks!